### ISSUE TITLE
[Feature] active icon in the norification area

### ISSUE THREAD

**Issue by gilbh** [pit-ray/win-vind#73]
Hi,

I think it would be useful if the user is able to discern the current mode through icon shape change in the notification area. Many similar background apps use this technique, add a short colorful indication to the app icon as indicator of current state (e.g., Dropbox would have a small blue circle indicating active sync at the moment, etc.).

The indication should not be complicated. A basic small visual indication on the app icon "V" (such as  a small letter indicating the mode? or a circle? a check mark? exclamation mark?) -- the indication should have a color-scale for which each mode is connected to (as in: blue for EDIT_NORMAL, red for INSERT, and so on -- just a random illustration).

Thanks 
Gil

**Comment by pit-ray** [comment]
That's the approach taken by [rcmdnk/vim_ahk](https://github.com/rcmdnk/vim_ahk).

However, as I have stated [here](https://github.com/pit-ray/win-vind/issues/31#issuecomment-898881614), even if implemented, it would be option.

To represent a mode, it must be a distinguishable symbol, but icons are only available in low resolutions such as 32x32. Therefore, I think it will be a messy look that almost hides the logo.

Thanks.

**Comment by pit-ray** [comment]
> I totally get it is not in your priority, but I think there would be huge value (at least to some people!). 

I see. I will consider the costs and its benefits. However, it will definitely be an option, not the default.

> A change of color would be the best/easiest to implement, with six(?) different colors (the default green on Insert?).

I think GUI, Editor, Insert, Resident is enough. What do you think?

Thanks.

**Comment by gilbh** [comment]
Yes, limiting to four variations (one of which is the default that you already have now) is a great idea.
